LayoutTests/TestExpectations

```diff
@@ -48,6 +48,7 @@ fast/forms/textarea/ios [ Skip ]
 fast/forms/watchos [ Skip ]
 fast/images/text-recognition [ Skip ]
 fast/images/mac [ Skip ]
+fast/images/spatial-image-controls.html [ Skip ]
```

Comment:
Wouldn't this mean the test is always skipped? We need to enable it on visionOS, right?

Reply:
In an earlier comment it was pointed out that EWS won't compile this until future SDK so it'll always fail on EWS, so it's been suggested offline that the test skip for now and later PR to un-skip on visionOS once EWS updates. Unless there's a better way?

Reply:
I see. That seems fine then.